my.css

```diff
@@ -0,0 +1,165 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header, .collection, {
+	z-index: 2;
+	position: relative;
+	color: white;
+}
+.header {
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)